tasks/embargo-check/embargo-check.yaml

```diff
@@ -59,12 +59,12 @@ spec:
         RC=0
 
         NUM_ISSUES=$(jq -cr '.releaseNotes.issues.fixed | length' "${DATA_FILE}")
-        for ((i = 0; i < $NUM_ISSUES; i++)); do
+        for ((i = 0; i < NUM_ISSUES; i++)); do
```

Comment:
missing `$`

Reply:
With "$" added, checkton reports the warning: $/${} is unnecessary on arithmetic variables. It's a fix for the warning.

Reply:
ok!